api/v1/helpers/verbs/doPost.js

```diff
@@ -97,6 +96,10 @@ function doPost(req, res, next, props) {
   }
 
   return postPromise.then((o) => {
+    if (!o) { // could be here from u.handleError
+      return;
```

Comment:
not quite. o is falsey when u.handleError is called, which means postPromise.then returns a falsey value.